test/fixtures/BootstrapCommand/extra/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "__TEST_VERSION__",
+  "version": "1.0.0",
+  "packages": [
+    {"glob": "packages/*/package.json"},
+    {"glob": "package-3/package.json"}
```

Comment:
@bclinkinbeard, you might find that #392 "helps" with one of the Appveyor failures (makes the test a little less platform-dependent).